mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -190,13 +190,13 @@ class Utils {
     if (gitHubMatch || gitLabMatch) {
       const baseUrl = gitHubMatch ? 'https://github.com/' : 'https://gitlab.com/';
       const match = gitHubMatch || gitLabMatch;
-      url = baseUrl + match[1] + '/' + match[2].replace(/.git/, '');
```

Comment:
Yes, this will fix the Regex! My approach was more about to not use the regex once the use case is very simple. I was curious about performance for such a simple use case, and just to let you know the regex usage is faster.

Checked test: Normal Replace x 8,974,544 ops/sec ±0.93% (65 runs sampled)
Checked test: Regex Replace x 13,693,190 ops/sec ±0.63% (64 runs sampled)

Nice! I am updating the PR. Thanks.

Reply:
Thanks for the update! `/\.git/g` looks like we're removing all `.git` accurences in `match`.

Reply:
I've added some more use cases. In this way, if you want to test different scenarios, let me know, and we can add there.